src/main/scala/module/SamRuleGenerator.scala

```diff
@@ -196,35 +196,71 @@ class SamRuleGenerator(var context: RuleTransducer.Context) extends RuleGenerato
-      // (add)
-	  
+      // *                                        (diag)
+      // *       ,---------,---------,---------,---------,
+      // *       |         |         |         |         |
+      // *       |         |    0    |    1    |    2    |
+      // *       |         |         |         |         |
+      // *       ;---------;---------;---------;---------;
+      // *       |         |         |         |         |
+      // * ortho |    3    |    4    |    5    |         |
+      // *       |         |         |         |         |
+      // *       '---------'---------'---------'---------'
+      // *       ,---------,---------,---------,---------,
+      // *       |         |         |         |    3    |
+      // *       |         |       14|        1|1        |
+      // *       |         |   15    |   113   |x5F591F00|
+      // *       ;---------;---------;---------;---------;
+      // *       |         |         |   113   |         |
+      // *       |2     111|111    11|11       |         |
+      // *       |         |         |         |         |
+      // *       '---------'---------'---------'---------'
+      /*
+      The following code for the large 45 curve is organized like so:
+      First, the overide from the orthogonal side to the diagonal side is described, ordered left to right.
+      Then, the the diagonal to orthogonal direction follows, ordered from right to left.
+      To to and from tiles are annotated in-line.  The necessary stability rules that apply to
+      that from and to tile combo (or its destabilized equivalent) follow indented thereafter.
+      */
+      //  ortho to diagonal (bottom row)
+      Rules += sam~WE | (Street ~> sam)~(2,0,111,0)             // ortho to 3
+      Rules += sam~(2,0,111,0) | (Street ~> sam)~(111,0,11,0)   // 3 to 4
+        Rules += sam~(2,0,111,0) | Street~(2,0,11,0) | % | sam~(111,0,11,0)
+      Rules += sam~(111,0,11,0) | (Street ~> sam)~(11,113,0,0)  // 4 to 5
+        Rules += sam~(111,0,11,0) | sam~(11,3,0,0) | % | sam~(11,113,0,0)
+        Rules += sam~(111,0,11,0) | Street~(11,3,0,0) | % | sam~(11,113,0,0)
+        Rules += sam~(111,0,11,0) | Street~(1,3,0,0) | % | sam~(11,113,0,0)
+      //  ortho to diagonal (bottom to top)
+      Rules += sam~(0,111,0,11) | (Street ~> sam)~(15,0,0,14)   // 4 to 0
+        Rules ++= stabilize(sam~(0,2,0,11) | Street~(0,0,0,0) | sam~(0,111,0,11) | sam~(15,0,0,14))
+        Rules ++= stabilize(sam~(0,2,0,11) | Street~(15,0,0,14) | sam~(0,111,0,11) | sam~(15,0,0,14))
+      Rules += sam~(0,11,113,0) | (Street ~> sam)~(113,0,0,1)   // 5 to 1
+        Rules += sam~(0,11,113,0) | Street~(3,0,0,1) | % | sam~(113,0,0,1)
+      //  ortho to diagonal (top row)
+      Rules += sam~(0,0,14,15) | (Street ~> sam)~(0,0,1,113)    // 0 to 1
+        Rules += sam~(0,0,14,15) | sam~(0,0,1,3) | % | sam~(0,0,1,113)
+      Rules += sam~(0,0,1,113) | IdTile(0x5F591F00,3,0,(Road~(1,3,0,0)).symmetries) | % | sam~(1,3,0,0)  // 1 to 2
```

Comment:
0x5F591F00 is the diagonal start of the 2×4 Street 45° curve. It doesn't actually have symmetries.

Though, removing the symmetries could result in mirrored occurences of `sam~(1,3,0,0)` in some instances, which is not ideal either. Not sure what to do. Maybe shrink the Street curve to the same 2×3 footprint SAM seems to use?

Reply:
Here are the rules for diagonal start of the Street 45 curve along with the SAM metarules RUL2:
![image](https://github.com/NAMTeam/Network-Addon-Mod/assets/26899145/5a016c85-90e5-4a2e-8faf-c66835959f77)
If I change 0x5F591F00 to `noSymmetries`, the last rule would be omitted.  Could the equivalent street rule also be omitted?  Note also that I've changed the original rule in this PR so that it doesn't use a mirrored tile.

Reply:
The footprint of the street 45 curve is easily shrunk.  I see no reason not to just do away with the extra tile.  You were correct that the noSymmetries version produces mirrored diagonal SAM tiles.  There are other rules contributing to this problem currently, but it would be good to eliminate this one anyway.